backend/utils/emails/newOrder.js

```diff
@@ -113,6 +115,16 @@ async function sendNewOrderEmail({
     ]
   }
 
+  // Generate the link for the buyer to see their order.
+  // If the order was recorded on the marketplace, the link uses the transaction hash.
+  // Otherwise the link uses the IPFS hash of the encrypted data.
+  const orderUrl = order.offerId
+    ? `${publicURL}/order/${cart.tx}?auth=${cart.dataKey}`
+    : `${publicURL}/order/${order.encryptedIpfsHash}?auth=${cart.dataKey}`
+
+  // Link for the merchant to the orders admin page.
+  const orderUrlAdmin = `${networkConfig.backendUrl}/admin/orders/${orderId}`
```

Comment:
this might need to be `${networkConfig.backendUrl}/#/admin/orders/${orderId}` (with an extra `#/`)

Reply:
ah yes, good catch!